src/components/Checkbox/tests/Checkbox.test.tsx

```diff
@@ -52,57 +39,25 @@ describe('<Checkbox />', () => {
       );
 
       (element.find('input')!.domNode as HTMLInputElement).checked = true;
-      element.find(Choice)?.trigger('onClick');
-
-      expect(spy).toHaveBeenCalledWith(false, 'MyCheckbox');
-    });
-
-    it('is called when space is pressed', () => {
-      const spy = jest.fn();
-      const element = mountWithApp(
-        <Checkbox id="MyCheckbox" label="Checkbox" onChange={spy} />,
-      );
-
-      element.find('input')!.trigger('onKeyUp', {
-        keyCode: Key.Space,
-      });
-
-      expect(spy).toHaveBeenCalledTimes(1);
-    });
```

Comment:
Since we no longer manually handler the keyUp events, we no longer need to test this.

Reply:
See https://github.com/Shopify/polaris-react/pull/4631#discussion_r755273209